diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/DiktatRuleSetProvider.kt

```diff
@@ -115,8 +112,7 @@ class DiktatRuleSetProvider(private var diktatConfigFile: String = DIKTAT_ANALYS
         log.debug("Will run $DIKTAT_RULE_SET_ID with $diktatConfigFile" +
                 " (it can be placed to the run directory or the default file from resources will be used)")
         val configPath = possibleConfigs
-            .filterNotNull()
```

Comment:
I love this one more actually, but leave your version

Reply:
This fixes issue with logging `null`s instead of names of non-existing files

Reply:
oh, I missed that! awesome